druid/druid-api/src/configs/config.ts

```diff
@@ -0,0 +1,65 @@
+export const config = {
+    apiEndPoint: "/druid/v2/",
+    apiPort: 8082,
+    druidEndPoint: "/druid/v2/",
+    druidHost: "http://localhost",
+    druidPort: 8082,
```

Comment:
The druidHost and port configurations should come from a env variable populated by ansible